SFTPStream.md

```diff
@@ -7,32 +7,54 @@ SFTPStream events
 
 **Server-only events**
 
+In all cases, responses to the various events **must** include the request's `reqID` as
+their first parameter. The `sftpStream` that the event fired on is the same object 
+that the various response methods should be invoked using.
```

Comment:
This last sentence sounds a little awkward to me. Maybe it should instead say something like:

```
Responses are sent using one of the `Server-only methods` listed further in this document.
```